### PR TITLE
public document

### PR DESCRIPTION
**Pull request by zMingGit**

### CODE REVIEW

**Comment by shoeper** [comment]
Without https://github.com/haiwen/python-seafile/pull/5/files the library does not work.

File: doc.md [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+
+**Return Type**
+
+A SeafFile Object
+
+**Exception**
+
+* File does not exist.
+
+### <a id="repo_get_dir"></a> Get Dir ###
```
Comment (freeplant) [inline doc.md]: 这里应该演示怎么列出目录的内容。需要把拿到 dir 对象后怎么打印内容补充上。

File: doc.md [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+	
+	client = seafileapi.connect('http://127.0.0.1:8000', 'test@admin.com', 'password')
+	repo_list = client.repos.list_repos()
+	
+
+**Return Type**
+
+A list of Repo Object
+
+### <a id="repos_create_repo"></a> Create Repo ###
```
Comment (freeplant) [inline doc.md]: Repo description 不需要了

File: doc.md [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+
+**Return Type**
+
+A SeafFile Object of upload file
+
+**Exception**
+
+* Local file does not exist.
+
+### <a id="seafdir_delete"></a> Delete SeafDir ###
```
Comment (freeplant) [inline doc.md]: 改成 delete a file or folder

File: doc.md [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+	repo = client.repos.get_repo('09c16e2a-ff1a-4207-99f3-1351c3f1e507')
+	seaffile = repo.get_file('/root/test.md')
+	
+	content = seaffile.get_content()
+
+**Return Type**
+
+File Content
+
+### <a id="seaffile_delete"></a> Delete SeafFile ###
```
Comment (freeplant) [inline doc.md]: Delete a file

Comment (freeplant) [reply doc.md]: SeafFile 是内部对象，写文档的时候应该从使用者的角度来写。对使用者来说，看到的是目录、文件、资料库。

**Comment by zMingGit** [comment]
@shoeper This is my fault. When I use python-seafile I get error, so I checked it and found some code has expired so I update it. And did not check the PR.

Thanks for the reminder.

**Pull request merged**